src/pages/home/HeaderView.js

```diff
@@ -93,6 +94,9 @@ const HeaderView = (props) => {
     const subtitle = getDefaultRoomSubtitle(props.report, props.policies);
     const isConcierge = participants.length === 1 && participants.includes(CONST.EMAIL.CONCIERGE);
 
+    // Show call button when there 1:1 chat with concierge. Else participants should not include automated expensify emails
```

Comment:
This comment would be more valuable if it explained _why_ we do not want to include automated Expensify emails. Otherwise, it just explains what the code does (which is not very valuable).